l10n_it_delivery_note/migrations/13.0.1.0.0/pre-migrate.py

```diff
@@ -29,7 +29,7 @@ def migrate(env, version):
         SET
             invoice_id = am.id
         FROM account_invoice inv
-            JOIN account_move am ON am.id = inv.move_id
```

Comment:
Il campo `invoice_id` viene aggiunto da [OpenUpgrade](https://github.com/OCA/OpenUpgrade/blob/45965d71b7cbc3ee4dd011567384edc5e66e3811/addons/account/migrations/13.0.1.1/pre-migration.py#L251). Questo significa che questo script non funzionerebbe nel caso in cui il db venisse migrato con https://upgrade.odoo.com
Sarei per supportarli entrambi analogamente a quanto era stato fatto [qui](https://github.com/OCA/l10n-italy/blob/ecc46d0fe8fa3110bb8ff6706906a1a0d5da2080/l10n_it_fatturapa/migrations/13.0.1.0.0/post-migrate.py#L67-L127).

Reply:
Io ho migrato il DB con https://upgrade.odoo.com e poi ho lanciato -u all con le addons di OCA e con lo script attuale non funzionava, con la modifica sì.